96.md

```diff
@@ -275,6 +272,43 @@ The successful response is a 200 OK one with just basic JSON fields:
 }
 ```
 
+## Listing files
+
+`GET $api_url?page=x&count=y`
```

Comment:
Cool but if client asks for page=1&count=20 and the server returns 10 items, the client doesn't know if there existed just 10 items or if the server max page size (count) is 10. So client won't know if it should check for next page or not.

One way of solving this is if the response is a jsob object instead of json array, something like this:
```js
{
  count: 10, // page size, not necessarily the number of items returned, which may be lower
  data: [
    { "id": ...,  ... },
    { "id": ...,  ... },
   // ...
  ]
}
```

This way client will know the max page size is 10 instead of 20.

Reply:
@arthurfranca look ok now?

Reply:
perfect